### ISSUE TITLE
Reducing tooltips for accessibility, and updating document links

### ISSUE THREAD

**Issue by sangwoohaan** [microsoft/BotFramework-Composer#7352]
## Describe the bug

We have recently discovered that tooltips will cause accessibility issues for screen readers. While we cannot completely remove tooltips altogether immediately, I want to start with the most complicated area which is Response editor. 

## Browser

- [ ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ ] Windows
- [ ] Ubuntu

## Expected behavior

![Screen Shot 2021-04-23 at 4 22 48 PM](https://user-images.githubusercontent.com/50963385/115938608-28931980-a450-11eb-8a1d-ad4eabe60cca.png)
![lglglg](https://user-images.githubusercontent.com/50963385/115938614-2d57cd80-a450-11eb-8bd2-7fdd26140f8b.png)

There are roughly three kinds of changes
1. Get the tooltips out visible as inline texts
2. Update document links
3. As a result of removing tooltips, dropdown menus have been changed to pop-up dialogs.

Here's the list of all the updates :
https://www.figma.com/file/q76DseJ7PK0g7zI1gXLzlD/SH_Language-Generation?node-id=607%3A1458

The scope of this change is limited to the Response editor.

**Comment by sangwoohaan** [comment]
@GeoffCoxMSFT @cwhitten Let me know if we can get this in for BUILD!

**Comment by sangwoohaan** [comment]
Related issue https://github.com/microsoft/BotFramework-Composer/issues/7085

**Comment by cwhitten** [comment]
@sangwoohaan what is the accessibility issues with the tooltips?
